src/scripts/getTemp.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/prefer-default-export */
+async function getIp() {
+  const ip = await fetch('https://api.ipify.org/?format=json');
```

Comment:
- [ ] You still need to have error handling even when working with async/await. 
[Please read this article](https://itnext.io/error-handling-with-async-await-in-js-26c3f20bc06a) on try/catch blocks and use them to handle errors for all the fetches you have in this file.